README.md

```diff
@@ -52,7 +53,7 @@ javascript
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://ameerhamzakhalid.github.io/todo-list/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] OPTIONAL
Kindly add instructions in your ReadMe under the _Getting Started_ section, on how to run the test cases for your project.